ext/gtest/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 2.8.8)
+project(gtest_builder C CXX)
+include(ExternalProject)
+
+#set(GTEST_FORCE_SHARED_CRT ON)
+#set(GTEST_DISABLE_PTHREADS OFF)
+
+ExternalProject_Add(googletest
```

Comment:
Hmm - don't see anywhere where it is on the next line.  Trying to standardize on keeping the target(s) on the first line.